### PR TITLE
[WFLY-13208] adds proposal for the Microprofile BOM

### PR DESCRIPTION
**Pull request by emmartins**
Issue: https://issues.jboss.org/browse/WFLY-13208

### CODE REVIEW

**Comment by rhusar** [comment]
There were question about artifacts should actually be included in the MicroProfile BOM, perhaps this should be described here as well to foster discussion.

**Comment by emmartins** [comment]
Updated to indicate the artifactId for EAP at R50

File: microprofile/WFLY-13208_Microprofile_BOM.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+* NONE
+
+== Test Plan
+
+* It should be tested that the Microprofile BOM is included in every WildFly/EAP build, and manages only the expected artifacts.
+
+== Community Documentation
+
+* The new BOM should be added to the WildFly/EAP documentation with respect to the BOMs.
```
Comment (tommaso-borgato) [inline microprofile/WFLY-13208_Microprofile_BOM.adoc:68]: @emmartins @dfitzmau could you please provide details with regard to the ongoing discussion here: https://issues.redhat.com/browse/EAP7-1459?focusedCommentId=14117375&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14117375

Comment (tommaso-borgato) [reply microprofile/WFLY-13208_Microprofile_BOM.adoc:68]: see https://issues.redhat.com/browse/EAP7-1459?focusedCommentId=14360868&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14360868

Comment (emmartins) [reply microprofile/WFLY-13208_Microprofile_BOM.adoc]: fixed

**Pull request merged**